src/CardsList.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useContext, useMemo } from 'react';
+
+import { useFormContext } from 'react-hook-form';
+
+import Grid from '@mui/material/Grid';
+
+import Cards from './Cards';
+import aparts from './fixtures/data';
+
+import Context from './Context';
+
+function CardsList() {
+  const { getValues } = useFormContext();
+
+  const ApartItem = getValues().currentApartment;
+  const RoomItem = getValues().currentRoomQuant;
+  const MinPrice = getValues().fromPrice;
+  const MaxPrice = getValues().lastPrice;
+  const CurrencyItem = getValues().currency;
+
+  const {
+    firstCardIndex, lastCardIndex, isFormSubmitted, cardListLength, setLength,
+  } = useContext(Context);
+
+  const CURRENCY = {
+    1: 'BYN',
+    2: '$',
+    3: '€',
+  };
+  const val = CurrencyItem;
```

Comment:
This also can be moved into function scope, no sense to store it in component scope